test/semanticValidation/specification/validateOperation/MULTIPLE_BODY_PARAMETERS/multipartMixed.json

```diff
@@ -0,0 +1,39 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Title",
+    "version": "1.0"
+  },
+  "paths": {
+    "/": {
+      "post": {
+        "operationId": "test",
+        "consumes": [
+          "multipart/mixed"
+        ],
+        "produces": [
+          "application/json"
```

Comment:
Would it matter if the produces is also multipart/mixed?

Reply:
I don't think so, but I'm happy to change the test to match.